lib/genevalidator/ext/array.rb

```diff
@@ -21,7 +21,7 @@ def mode
     end
 
     def sample_variance
-      m   = mean
+      m   = inject(:+).to_f / size
```

Comment:
Why not define `mean` on Array instead of calling `inject(:+).to_f / size` several times?